README.md

```diff
@@ -0,0 +1,3 @@
+# OPC XML-DA client in Haskell
+
+This library provide the client for OPC XML-DA protocol whose specification can be found [here](http://www.diit.unict.it/users/scava/dispense/II/OPCDataAccessXMLSpecification.pdf)
```

Comment:
There's a few translation mistakes. Here's what I suggest.

```markdown
# OPC XML-DA client for Haskell

An implementation of OPC XML-DA protocol for client applications. The specification for the protocol can be found [here](http://www.diit.unict.it/users/scava/dispense/II/OPCDataAccessXMLSpecification.pdf).
```

Reply:
Thanks, fixed it)